python/aswfdocker/builder.py

```diff
@@ -51,10 +57,10 @@ def make_bake_dict(self) -> typing.Dict[str, dict]:
                     if version in versions_to_bake:
                         # Only one version per image needed
                         continue
-                    if version_info.ci_common_version != major_version:
-                        # Only bake images for ci_common!
-                        version = version_info.ci_common_version
-                        major_version = utils.get_major_version(version)
+                    # if version_info.ci_common_version != major_version:
+                    #    # Only bake conan images in ci_common container!
+                    #    version = version_info.ci_common_version
+                    #    major_version = utils.get_major_version(version)
```

Comment:
Should this whole block be removed?

Reply:
I went back and forth on this one several times, but I think now with the "building Conan packages in buildx" support, this is no longer valid, so I'll get rid of the block altogether.